src/i18n/fi.json

```diff
@@ -0,0 +1,160 @@
+{
+  "appName": "Profiili",
+  "berthErrors": {
+    "close": "Sulje",
+    "explanation": "Et voi poistaa tietojasi, koska Venepaikka-palvelu tarvitsee tietojasi laskutukseen. Ota yhteyttä Venepaikka-palvelun asiakaspalveluun irtisanoaksesi sopimuksesi ja yritä sen jälkeen poistoa uudelleen.",
+    "title": "Toimintoa ei voi suorittaa."
+  },
```

Comment:
This file contains some texts that probably aren't used in this demo.

Reply:
File removed.